web-console/knikubevirt/snapshots/snapshots.md

```diff
@@ -0,0 +1,75 @@
+# Virtual Machines Snapshots
+
+Snapshots allow users to create a copy of the VM disks and memory, mainly for restore and backup purposes. The snapshots are taken live or offline depending on if the VM is running or not. 
+This document describes the snapshot design flow.  
+
+
+
+
+## From VM List 
+
+![Offline VM list - snapshot action from a single VM kebab](img/1-00.jpg)
+When clicking on the kebab menu in the VM list, the user will be able to take a snapshot of the VM, meaning he will copy the disks, memory content, and metadata for restoring, backup, creating templates and cloning VMs.
+
+![Offline Snapshot modal - default](img/1-01.jpg)
+In the basic modal, the snapshot will take all current configuration and selected parameters.
+The user must add a descriptive name.
+![Offline Snapshot modal - selecting disks](img/1-02.jpg)
+In the disks field, the user will be able to decide if to include all disks or just some of them and if to include memory as part of the snapshot or not.
+![Offline Snapshot modal - disks customization](img/1-03.jpg)
+![Offline Snapshot modal - disks customized](img/1-04.jpg)
+
+![Offline Snapshotting status indication](img/1-05.jpg)
```

Comment:
Can we update this screen to remove the toast notification. I believe we won't be showing these types of toast (success creation) moving forward.

Reply:
Couldn't locate this underline...We're talking about the three pop-up images where the user selects disks?

Reply:
![IMG_2510](https://user-images.githubusercontent.com/9122899/58762737-bf831f80-8520-11e9-80be-b9ec9af1f771.jpeg)

I think we’re discussing this image, where “Snapshotting” is underlined. Underlines only appear when blue status text is hovered over with a mouse. You could add a pink hover hand here and the underline would be correct.

Reply:
Yup that's the one.